packages/kit/src/runtime/server/index.js

```diff
@@ -39,6 +39,10 @@ export async function respond(incoming, options, state = {}) {
 		...incoming,
 		headers,
 		body: parse_body(incoming.rawBody, headers),
+		rawBody:
+			typeof incoming.rawBody === 'string'
+				? new TextEncoder().encode(incoming.rawBody)
+				: incoming.rawBody,
```

Comment:
is there a case where `rawBody` could be a string (other than an out-of-date adapter, in which case we could/should throw an error), or could we simplify this?

```suggestion
		rawBody: incoming.rawBody,
```

Reply:
Right now these are the places where changes are necessary:

https://github.com/sveltejs/kit/blob/a24aba9120aa97e87d20b74ef7146ce737fb3b15/packages/kit/src/core/adapt/prerender.js#L153-L161
https://github.com/sveltejs/kit/blob/a24aba9120aa97e87d20b74ef7146ce737fb3b15/packages/kit/src/core/adapt/prerender.js#L285-L294
https://github.com/sveltejs/kit/blob/a24aba9120aa97e87d20b74ef7146ce737fb3b15/packages/kit/src/core/dev/index.js#L363-L371
https://github.com/sveltejs/kit/blob/a24aba9120aa97e87d20b74ef7146ce737fb3b15/packages/kit/src/runtime/server/page/load_node.js#L146-L154